screens/AccountCloseConfirmForm.js

```diff
@@ -0,0 +1,452 @@
+                  				mostrecentShift.close.closingShiftReport.totalByPaymentMethod.CASH.orderTotal
+												}
+                  		</Text>
+                			</View>
+              		</View>
+              	)}                        
+
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                  	Amount of payment
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Text>$&nbsp;?</Text>
+                	</View>
+              	</View>     
+
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	Amount of cash register should have
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Field
+              				name="name"
+              				component={InputText}
+              				//validate={isRequired}
+              				//placeholder={t('productName')}
+              				placeholder={"??"}
+              				secureTextEntry={false}
+              				height={35}
+            				/>
+                	</View>
+              	</View>
+
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	Amount actually owned by cash register
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Field
+              				name="name"
+              				component={InputText}
+              				//validate={isRequired}
+              				//placeholder={t('productName')}
+              				placeholder={"??"}
+              				secureTextEntry={false}
+              				height={35}
+            				/>
+                	</View>
+              	</View>
+            	</View>             		
+          	</View>
+        	{/* #Cash */}
+
+        	{/* Credit Card */}
+          	<View style={styles.sectionBar}>             
+            	<View style={[styles.tableCellView, {flex: 4}]}>
+              	<TouchableOpacity>
+                	<Text style={styles.sectionBarTextSmall}>
+                  	{t('card')}
+                	</Text>
+              	</TouchableOpacity>
+            	</View>
+          	</View>
+
+          	<View style={[styles.container, styles.no_mgrTop]}>
+            	<View style={{flex: 3, justifyContent: 'center'}}>              	              
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	Amount of cash register should have
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Field
```

Comment:
This shouldn't be text field. This should display the total credit card sales.

Reply:
mostrecentShift.close.closingShiftReport.totalByPaymentMethod.CARD.orderTotal